src/main/java/frc/robot/subsystems/drivetrain/commands/TurnInPlace.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot.subsystems.drivetrain.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.OI;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+import frc.robot.subsystems.drivetrain.SwerveDrive;
+import frc.robot.valuetuner.WebConstant;
+import org.techfire225.webapp.FireLog;
+
+public class TurnInPlace extends CommandBase {
+
+    private SwerveDrive swerveDrive;
+    //    private WebConstant target = new WebConstant("targetAngle", 0);
+    private WebConstant target0 = new WebConstant("target0", 0);
+    private WebConstant target1 = new WebConstant("target1", 0);
+    private WebConstant target2 = new WebConstant("target2", 0);
+    private WebConstant target3 = new WebConstant("target3", 0);
```

Comment:
```suggestion
    private final SwerveDrive swerveDrive;
    private final WebConstant target0 = new WebConstant("target0", 0);
    private final WebConstant target1 = new WebConstant("target1", 0);
    private final WebConstant target2 = new WebConstant("target2", 0);
    private final WebConstant target3 = new WebConstant("target3", 0);
```

and what is target0, target1, etc...

Reply:
target for each wheel numbered by index

Reply:
I would appreciate it if you will add a comment that explains that